src/components/UI/ResourcesLink/ResourcesLink.tsx

```diff
@@ -0,0 +1,76 @@
+import styled from "@emotion/styled";
+import React, { FunctionComponent } from "react";
+import { Download } from "react-feather";
+import { mixin, vars } from "../../../styles";
+
+interface ResourcesLinkProps {
+  className?: string;
+  title: string;
+  details: {
+    description: string;
+    url: string;
+    icon?: string;
+  };
+  type: string;
+}
+
+const ResourcesLinkUnstyled: FunctionComponent<ResourcesLinkProps> = ({ className, title, details, type }) => {
+  return (
+    <div className={`${className}`}>
+      <div className="link-wrapper">
+        <div className="title">{title}</div>
+        <div className="link-container">
+          {type === "link" ? (
+            <a className="link" href={details.url} target="_blank" rel="noopener noreferrer" data-testid="link">
+              {details.description}
+            </a>
+          ) : (
+            <div className="download-wrapper">
+              <a href={details.url} download={`${details.description}.pdf`} className="link" data-testid="download">
+                <Download className="mr-1" />
+                {details.description}
+              </a>
+            </div>
+          )}
```

Comment:
maybe check for the type directly without the `else` condition

Reply:
ok